cmd/signer/main.go

```diff
@@ -0,0 +1,643 @@
+// Copyright 2017 The go-ethereum Authors
```

Comment:
2018

Reply:
Are you sure? I crated the file(s) in 2017